packages/prbnmcn-dagger-gsl/prbnmcn-dagger-gsl.0.0.1/opam

```diff
@@ -0,0 +1,37 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Probabilistic programming library: GSL-based samplers"
+description: "dagger probabilistic programming library: GSL samplers"
+maintainer: ["igarnier@protonmail.com"]
+authors: ["Ilias Garnier"]
+license: "GPL-3.0-or-later"
+homepage: "http://github.com/igarnier/prbnmcn-dagger"
+bug-reports: "http://github.com/igarnier/prbnmcn-dagger"
+depends: [
+  "dune" {>= "2.8" & >= "2.8"}
```

Comment:
```suggestion
  "dune" {>= "2.8"}
```
This is automatically fixed removing dune from the dependencies in the dune-project file (dune is added automatically in any case)